train.py

```diff
@@ -99,7 +100,7 @@
     "--rollout.num_cores", dest="num_cores", type=int, default=None,
       help="number of cores to use for training (default: num_envs)")
   parser.add_argument(
-    "--rollout.num_envs", dest="num_envs", type=int, default=4,
```

Comment:
Rest of them as well. If it's helpful for testing, you can save these args to a train_debug.sh script